src/Fixer/Import/FullyQualifiedStrictTypesFixer.php

```diff
@@ -188,6 +188,34 @@ protected function createConfigurationDefinition(): FixerConfigurationResolverIn
                 ->setAllowedTypes(['bool'])
                 ->setDefault(false)
                 ->getOption(),
+            (new FixerOptionBuilder(
+                'phpdoc_tags',
+                'Collection of PHPDoc annotation tags where FQCNs should be processed. As of now only simple tags with `@tag \F\Q\C\N` format are supported (no complex types).'
+            ))
+                ->setAllowedTypes(['array'])
+                ->setDefault([
+                    'param',
+                    'phpstan-param',
+                    'phpstan-property',
+                    'phpstan-property-read',
+                    'phpstan-property-write',
```

Comment:
what would be the case of `phpstan-property-read` enabled, but `phpstan-property-write` disabled?

imagine someone actually use the config, phpstan introducing new annotation - they need to manually add the annotation here.
I would rather have a group of annotations like `@phpstan` or `phpstan*` for all phpstan-based annotations

(same for others)

Reply:
There are more `phpstan-*` and `psalm-*` annotations, but these have different purpose. I only included these related to types, tool-specific variants of regular phpDoc ones. I doubt new annotations will emerge 🙂.

Maybe we can focus on regular phpDoc tags, and automatically add prefixed ones 🤔? Like when someone includes `param` and automatically `phpstan-param` and `psalm-param` are included. It would be more logical to me than wildcards, because you may want to process `param` but not `var` (prefixed variants). The other alternative is to ignore tool-specific variants in default config and just let people configure it by themselves, but as far as I remember other fixers handle these too.

Reply:
like, tbh I see only 2 cases myself - want to process annotations (any of them) or not. boolean config - do we have request to go granular and support some, but not all annotations?

Reply:
For example `@see` is something that I think can be processed, and there was a voice it shouldn't be. Also, having this configurable makes it possible to clear large codebases step by step (by enabling one tag after another and providing partial diffs). List of tags was previously hardcoded and there was a request to add new tags - having it configurable allows people to do it in a flexible way, without changing the fixer's code.